Makefile

```diff
@@ -0,0 +1,3 @@
+test:
```

Comment:
Do we really need this file? I always run go test --count=1 ./... and it does the same job.

Reply:
yeah, it is not really required. I have removed it and updated the PR with changes requested.